src/components/ComplaintDetail/ComplaintDetailBody.js

```diff
@@ -57,7 +57,7 @@ ConsumerConsent.propTypes = { value: PropTypes.string };
 
 const CompanyTimely = ({ value }) => {
   if (!value) {
-    return null;
+    return 'N/A';
```

Comment:
Maybe wrap this in a div (or the same `<span className="body-copy">` element that the other fields use) so that it gets some margin underneath it:

<img width="392" alt="image" src="https://github.com/cfpb/ccdb5-ui/assets/1060248/650c4a4d-7d62-4d64-97c2-44d664ef2b7e">